src/proposal/repost.rs

```diff
@@ -0,0 +1,43 @@
+use near_sdk::serde_json::json;
+use near_sdk::{env, AccountId, Promise};
+
+use crate::{get_proposal_description, social_db::social_db_contract, Proposal};
+
+fn repost_internal(proposal: Proposal, contract_address: AccountId) -> near_sdk::serde_json::Value {
+    let proposal_link = format!("/devgovgigs.near/widget/Proposal?id={}", proposal.id);
```

Comment:
@Megha-Dev-19 What should be the link to the Proposal? `/devhub.near/widget/app?page=proposal&id={}`?

Reply:
Yes that’s correct 👍